dataclasses_json/mm.py

```diff
@@ -282,7 +282,7 @@ def build_schema(cls: typing.Type[A],
                                  if field.name != 'dataclass_json_config')})
 
     @post_load
-    def make_instance(self, kvs):
+    def make_instance(self, kvs, **kwargs):
```

Comment:
I assume `**kwargs` is intentionally ignored as part of the fix?

Reply:
Yes seems to be the same fix described here: https://github.com/marshmallow-code/flask-marshmallow/issues/139